src/components/modals/ModalTest/ModalTest.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Box, Text } from '@makerdao/ui-components-core';
+
+const ModalTest = ({ dispatch, onClose }) => {
+  return (
+    <Box maxWidth="71.8rem">
+      <Text.h2 textAlign="center" mb="xl">
```

Comment:
Looks like `Text.h2` is only available in more recent versions of `ui-components-core`.